docs/how-tos/howto-stake-unstake.md

```diff
@@ -0,0 +1,130 @@
+
+## Staking Smart Contract
+
+The repo for the Staking Smart Contract is located [here](https://github.com/0xPolygon/staking-contracts).
+
+It holds the necessary testing scripts, ABI files and most importantly the Staking Smart Contract itself.
+
+## Setting up an N node cluster
+
+Setting up a cluster with the Polygon SDK is covered in the [setup IBFT locally](/docs/how-tos/howto-setup-ibft/howto-set-ibft-locally)
+/ [setup IBFT on the cloud](/docs/how-tos/howto-setup-ibft/howto-set-ibft-on-the-cloud) sections.
+
+The **only difference** between setting up a PoS and PoA cluster is in the genesis generation part.
+
+**When generating the genesis file for a PoS cluster, an additional flag is needed `--pos`**:
+```bash
+go run main.go genesis --pos ...
+```
+
+## Setting the length of an epoch
+
+Epochs are covered in detail in the [Epoch Blocks](/docs/guides/pos-concepts#epoch-blocks) section.
+
+To set the size of an epoch for a cluster (in blocks), when generating the genesis file, an additional flag is 
+specificed `--epoch-size`:
+```bash
+go run main.go genesis --epoch-size 50
+```
+This value specifies in the genesis file that the epoch size should be `50` blocks.
+
+The default value for the size of an epoch (in blocks) is `100000`.
+
+## Using the Staking Smart Contract scripts
+### Prerequisites
+The Staking Smart Contract repo is a Hardhat project, which requires NPM. 
+
+To initialize it correctly, in the main directory run:
+```bash
+npm install
+````
+
+### Setting up the provided helper scripts
+
+Scripts for interacting with the deployed Staking Smart Contract are located on the [Staking Smart Contract repo](https://github.com/0xPolygon/staking-contracts).
+
+Create an `.env` file with the following parameters in the Smart Contracts repo location:
+```bash
+POLYGON_SDK_JSONRPC_URL=http://localhost:10002
+PRIVATE_KEYS=0x0454f3ec51e7d6971fc345998bb2ba483a8d9d30d46ad890434e6f88ecb97544
+STAKING_CONTRACT_ADDRESS=0x0000000000000000000000000000000000001001
+```
+
+Where the parameters are:
+* **POLYGON_SDK_JSONRPC_URL** - the JSON-RPC endpoint for the running node
+* **PRIVATE_KEYS** - private keys of the staker address
+* **STAKING_CONTRACT_ADDRESS** - the address of the staking smart contract (default `0x0000000000000000000000000000000000001001`)
+
+### Staking funds
+
+:::info Staking address
+The Staking Smart Contract is pre-deployed at address `0x0000000000000000000000000000000000001001`.
+
+Any kind of interaction with the staking mechanism is done through the Staking Smart Contract at the specified
+address.
+
+To learn more about the Staking Smart Contract, please visit the [Staking Smart Contract](/docs/guides/pos-concepts#contract-pre-deployment) 
+section.
+:::
+
+In order to become part of the validator set, an address needs to stake a certain amount of funds above a threshold.
+
+Currently, the default threshold for becoming part of the validator set is `1 ETH`.
```

Comment:
This threshold is fixed? If we say that the default value is 1ETH then it's implied that there's a way to change the default value that's not documented.

Reply:
Currently, we haven't added support for specifying a staking threshold, since it's hard-coded into the SC:
https://github.com/0xPolygon/staking-contracts/blob/2ddb6f9afaf7ebfd536a81b252d3c2b77e6a6f02/contracts/staking.sol#L7

Reply:
I've updated the phrasing to be more clear 👍 

901e07d